pyproject.toml

```diff
@@ -12,6 +12,8 @@ cookiecutter = "^1.7.2"
 [tool.poetry.dev-dependencies]
 flake8 = "^3.8.4"
 black = "^20.8b1"
+poetry = "^1.1.5"
```

Comment:
It looks weird that poetry is a dependency since it's needed to install the deps?